pages.zh/common/2to3.md

```diff
@@ -0,0 +1,32 @@
+# 2to3
+
+> 自动将 Python 2 代码转换成 Python 3。
+> 更多信息：<https://docs.python.org/3/library/2to3.html>.
+
+- 显示将执行的变动但不执行（dry-run）：
+
+`2to3 {{文件.py}}`
+
+- 将 Python 2 文件转化为 Python 3：
```

Comment:
```suggestion
- 将 Python 2 文件转化为 Python 3:
```

Please use half-width when the last character of a sentence is number or Latin character. I'm sorry that I forgot to mention this rule in style guide. I think this is the very last suggestion, the page could be merged after fixing them, thank you.

Reply:
ok, I will follow the written rules